code/datums/components/food/edible.dm

```diff
@@ -65,6 +79,29 @@ Behavior that's still missing from this component that original food items had t
 			else
 				owner.reagents.add_reagent(rid, amount)
 
+/datum/component/edible/InheritComponent(datum/component/C,
+	i_am_original,
+	list/initial_reagents,
+	food_flags = NONE,
+	foodtypes = NONE,
+	volume = 50,
+	eat_time = 30,
+	list/tastes,
+	list/eatverbs = list("bite","chew","nibble","gnaw","gobble","chomp"),
+	bite_consumption = 2,
+	datum/callback/after_eat,
+	datum/callback/on_consume)
```

Comment:
Can you put the parenthese on its own line? Makes it clearer from a skim where the actual code begins.

Also, this is a terrifying function signature,